src/views/product-tutorials-view/helpers/page-data/index.ts

```diff
@@ -0,0 +1,42 @@
+import { TableOfContentsHeading } from 'layouts/sidebar-sidecar/components/table-of-contents'
+import { PageSlugOption } from 'lib/learn-client/api/page'
+import { HeroHeadingVisualProps } from 'views/product-landing/components/hero-heading-visual/types'
+import { OverviewCtaProps } from 'views/product-landing/components/overview-cta/types'
+import { ProductViewBlock } from 'views/product-tutorials-view/components/product-view-content'
+import getProductPageContent from './get-product-page-content'
+import { processPageData } from './process-page-data'
+
+/**
+ * Fetch and process page data for a Tutorial product landing page.
+ */
+async function getProcessedPageData(
```

Comment:
Consolidated some repetition between the props fetching for:
- `/<product>/tutorials`
- `/hcp/tutorials/`
- `/well-architected-framework`
All three now use this `getProcessedPageData`.